test/common.test.js

```diff
@@ -132,4 +132,17 @@ describe('OrbitDB CLI - Common', function () {
       assert.equal(err, 'Error: No input data!')
     })
   })
+
+  describe('version', () => {
+    it('shows current version of orbit db', () => {
+      const version = CLI(`version`)
+      assert.equal(version.toString().replace('\n', ''),'0.0.8')
```

Comment:
Let's read the version number from `package.json`, just like in the implementation. That way we don't have to update the test every time we publish a new version on npm :)

Otherswise this is perfect, great job! 👍

Reply:
Well I wanted to include the `package.json` to read the version number as I did while writing the version command. The purpose I hard coded it was there is same code to read the version number in both cases, so I felt it was beating the purpose. The downside would be to update the test every time you release a new version.

If you suggest I can read the `package.json` and check it over here, but it would be just mocking the behavior we added test for. It's your call to take, I will update it as you suggest.

Reply:
The important thing here to test is to make sure a) the CLI command returns a version number and b) the version number matches the package version (this will make sure that version number *is* updated with a new release). I would prefer to read it from package.json automatically.

Reply:
:+1: Agreed. Will make the changes as required.